src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (Object.keys(inverse).includes(value.toString())) {
```

Comment:
redundanr to such big checking
it can be 
`if (value in inverese) {` or `if (inverse[value]) {` or `if (inverse.hasOwnProperty(value)) {`